react/components/Alert/index.test.js

```diff
@@ -3,7 +3,7 @@ import { render, fireEvent } from 'react-testing-library'
 
 import Alert from './index'
 
-describe('Alert', () => {
+describe.skip('Alert', () => {
```

Comment:
Why the skip?

Reply:
![image](https://user-images.githubusercontent.com/284515/76220912-04965980-61f7-11ea-9c22-293f0d5d1dbe.png)

and

![image](https://user-images.githubusercontent.com/284515/76220946-111ab200-61f7-11ea-821c-28aa817fe066.png)

Reply:
There should be a comment above the skip then

Reply:
Add a comment above the skip